validate/validate.go

```diff
@@ -384,6 +384,18 @@ func (v *Validator) CheckLinux() (msgs []string) {
 		msgs = append(msgs, "rootfsPropagation must be empty or one of \"private|rprivate|slave|rslave|shared|rshared\"")
 	}
 
+	for _, maskedPath := range v.spec.Linux.MaskedPaths {
+		if !filepath.IsAbs(maskedPath) {
```

Comment:
maskedPath and readonlyPath are Linux-specific, I think we don't need cross-platform validation here.